src/national.py

```diff
@@ -57,15 +68,44 @@ def get_national_forecast(
     logger.debug("Get national forecasts")
 
     logger.debug("Getting forecast.")
-    national_forecast_values = get_latest_forecast_values_for_a_specific_gsp_from_database(
+    forecast_values = get_latest_forecast_values_for_a_specific_gsp_from_database(
         session=session, gsp_id=0, forecast_horizon_minutes=forecast_horizon_minutes
     )
 
     logger.debug(
-        f"Got national forecasts with {len(national_forecast_values)} forecast values. "
+        f"Got national forecasts with {len(forecast_values)} forecast values. "
         f"Now adjusting by at most {adjust_limit} MW"
     )
-    national_forecast_values = [f.adjust(limit=adjust_limit) for f in national_forecast_values]
+
+    forecast_values = [f.adjust(limit=adjust_limit) for f in forecast_values]
+
+    # change to NationalForecastValue
+    national_forecast_values = []
+    for f in forecast_values:
+        # change to NationalForecastValue
+        plevels = f._properties
+        national_forecast_value = NationalForecastValue(**f.__dict__)
+        national_forecast_value.plevels = plevels
+
+        # add default values in, we will remove this at some point
+        if not isinstance(national_forecast_value.plevels, dict):
```

Comment:
Does this trigger on an empty dict as well as things that aren't dicts?

Reply:
hmm, thats a good one, I should add that case